SwiftScripts/Sources/TargetConfig/Module.swift

```diff
@@ -6,7 +6,7 @@ public struct Module {
 
   public init?(module: String) {
     switch module.lowercased() {
-    case "none": self.moduleType = .none
+    case "none": self.moduleType = .embeddedInTarget(name: "")
```

Comment:
Is a change away from `none` required here too?

Reply:
Not at the moment. We aren't using it, so really this could just be `fatalError()`. If we ever use it, we should fix this. But this is just our custom SwiftScripts that we use for generating OUR example schemas, so it's okay that it's not comprehensive.